v15-upgrade-log.md

```diff
@@ -72,3 +72,11 @@ Now we will upgrade to SpatialOS 15.0.0.
    json format requires all fields to be specified (even if they are empty).
 1. As a drive-by fix, we make sure we only attemp to send updates when we have
    authority.
+1. Fix up the launch configurations. We only have one, `default_launch.json`. As
```

Comment:
Ahh, this answers my earlier question about permissions for the physics worker... I don't think it's a huge deal but I think this might have made more sense to include along with the code to assign the partitions.

Reply:
I have instead changed the description on the step which implements the authority model to say that we will need to do this later.